### PR TITLE
fix issue #162

### PR DESCRIPTION
**Pull request by eyotang**
fix issue #162

### CODE REVIEW

**Comment by CarlJi** [comment]
Great contributions !!  I will review it soon and carefully.

File: pkg/cover/instrument.go [hunk 1]
```diff
@@ -225,6 +246,64 @@ func registerSelf(address string) ([]byte, error) {
...
+        }
+        req.Header.Set("Content-Type", "application/json")
+
+        resp, err := http.DefaultClient.Do(req)
+        if err != nil && isNetworkError(err) {
+                log.Printf("[goc][WARN]error occurred:%v, try again", err)
+                resp, err = http.DefaultClient.Do(req)
+        }
+        if err != nil {
+                return nil, fmt.Errorf("failed to register into coverage center, err:%v", err)
```
Comment (CarlJi) [inline pkg/cover/instrument.go:270]: typo. register => deregister from

File: pkg/cover/instrument.go [hunk 2]
```diff
@@ -225,6 +246,64 @@ func registerSelf(address string) ([]byte, error) {
...
+        }
+        defer resp.Body.Close()
+
+        body, err := ioutil.ReadAll(resp.Body)
+        if err != nil {
+                return nil, fmt.Errorf("failed to read response body, err:%v", err)
+        }
+
+        if resp.StatusCode != 200 {
+                err = fmt.Errorf("failed to register into coverage center, response code %d", resp.StatusCode)
```
Comment (CarlJi) [inline pkg/cover/instrument.go:280]: typo.  register into => deregister from

File: pkg/cover/instrument.go (hunk 1 above)
Comment (eyotang) [reply pkg/cover/instrument.go:270]: OK

**Pull request merged**